bridge.go

```diff
@@ -353,6 +353,8 @@ func (b *bridge) SyncSourceFor(id ProjectIdentifier) error {
 	return b.sm.SyncSourceFor(id)
 }
 
+func (b *bridge) Release() { b.sm.Release() }
```

Comment:
Yep, good points. Up until this issue, and #159, there wasn't a strong reason not to have `*bridge` implement `SourceManager` - but I've always been a bit dubious about it. That's why `sourceBridge` is a separate interface in the first place; it used to not compose `SourceManager` at all. This issue, and #159, are really the ones that expose the difference between what the bridge does and what a full-on `SourceManager` does - so, it's time to separate the interfaces. 

Having it panic was at least in part a an attempt to avoid a merge conflict with #210 by removing the interface composition...but since that's stalled until I can find the version type leak, it'd be fine to just do the separation here - stop composing `SourceManager` in `sourceBridge`, and instead include just the subset of methods (everything except `Release()`) that we actually need on the bridge directly.